plugins/FIC_Markdown.py

```diff
@@ -42,11 +42,11 @@ def _generate_repo_url(self, repository_type):
         repositories_data = json.load(self.load(None, REPOSITORIES_FILE))
         return repositories_data.get(repository_type).get(self.repo_name).get("url")
 
-    def no_data_for_md(self, repository_type, main_markdown=False, repo_markdown=False):
+    def no_data_for_md(self, argument, main_markdown=False, repo_markdown=False):
         if main_markdown:
-            return "No commits in the past {} days.. see the longer history in {} ".format(DEFAULT_DAYS, self._get_js_md_links(self.repo_name)[1])
+            return "No commits in the past {} days.. see the longer history ".format(DEFAULT_DAYS) + "[here](" + argument + ")" + "\n"
         elif repo_markdown:
-            return "No recent changes on this repository.. see the entire changelog by accessing this [link]({})".format(self._generate_repo_url(repository_type))
+            return "No recent changes on this repository.. see the entire changelog by accessing this [link]({})".format(self._generate_repo_url(argument))
```

Comment:
Here we should use string literals: f-strings (py3.6)
```python
        if main_markdown:
            return f"No commits in the past {DEFAULT_DAYS} days.. see the longer history [here]({argument})"
        elif repo_markdown:
            return f"No recent changes on this repository.. see the entire changelog by accessing this [link]({self._generate_repo_url(argument)})"
```
I'll add these changes in a separate PR.